subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -129,6 +135,15 @@ int lll_adv_reset(void)
 {
 	int err;
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+#if defined(CONFIG_BT_CTLR_ADV_AUX_SET)
```

Comment:
```suggestion
#if (BT_CTLR_ADV_AUX_SET > 0)
```

note, without `CONFIG_`

Reply:
@cvinayak fixed